package.json

```diff
@@ -41,9 +41,12 @@
     "typescript": "^3.9.7"
   },
   "dependencies": {
+    "@admin-bro/express": "^3.0.0-beta.3",
```

Comment:
this is something we have to discuss. Originally AdminBroExpress has these peerDependencies, so user has to install them on his own. If we left them as they were, in order to install @admin-bro/typeorm user will have to:

```
yarn add @admin-bro/typeorm @admin-bro/express express-formidable express-session admin-bro
```

I added them here so now:

```
yarn add @admin-bro/typeorm admin-bro
```

will be enough, but there is a error in the terminal that these peerDependencies will have to be installed. 

Basically - to discuss